pages/common/msmtp.md

```diff
@@ -0,0 +1,16 @@
+# msmtp
+
+> An SMTP client.
+> In the default sendmail mode, msmtp reads a mail from standard input and sendscit to an SMTP server for delivery.
```

Comment:
I feel this sentence should be shortened. Maybe cap it to 80 chars or so. Also, there's a typo: "sendscit" --> "sends it".